docs/development/usage.rst

```diff
@@ -64,6 +64,9 @@ There are a range of handy development functions that you might want to use to s
     * - Running ``pytest`` commands inside the ``poetry`` environment.
       - Make sure you have already installed ``tidy3d`` in ``poetry`` and you are in the root directory.
       - ``poetry run pytest``
+    * - Run ``coverage`` testing from the ``poetry`` environment.
+      -
```

Comment:
extra line? (and below)?

Reply:
So this is a list-table in rst so it's just an empty cell in the table

Reply:
ah ok got it.